docs/commcare-cloud/commands/index.md

```diff
@@ -415,9 +415,11 @@ Manage services.
 
 Usage examples:
    cchq <env> service postgresql status
+   cchq <env> service riakcs restart --only riak,riakcs
    cchq <env> service celery help
    cchq <env> service celery restart --limit <host>
    cchq <env> service celery restart --only <queue-name>
```

Comment:
I think you need to remove this line for command doc tests to pass (although there seems to be another real test failure)

Reply:
:+1: also had to fix something else. OK, I'm really done now. Going to bed.